.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,3 +3,35 @@ To be copied to the [draft changelog](https://github.com/DataBiosphere/toil/wiki
 
  * Whatsits now frobnicated
 
+## Reviewer Checklist
+
+<!-- To be kept in sync with docs/contributing/checklist.rst -->
+
+ * [ ] Make sure it is coming from `issues/XXXX-fix-the-thing` in the Toil repo, or from an external repo.
+    * [ ] If it is coming from an external repo, make sure to pull it in for CI with:
+        ```
+        contrib/admin/test-pr otheruser theirbranchname issues/XXXX-fix-the-thing
+        ```
+    * [ ] If there is no associated issue, [create one](https://github.com/DataBiosphere/toil/issues/new).
+* [ ] Read through the code changes. Make sure that it doesn't have:
+    * [ ] Addition of trailing whitespace.
+    * [ ] New variable or member names in `camelCase` that want to be in `snake_case`.
+    * [ ] New functions without [type hints](https://docs.python.org/3/library/typing.html).
+    * [ ] New functions or classes without informative docstrings.
+    * [ ] Changes to semantics not reflected in the relevant docstrings.
+    * [ ] New or changed command line options for Toil workflows that are not reflected in `docs/running/cliOptions.rst`
+    * [ ] New features without tests.
```

Comment:
Can be backed up with a coverage alert that combines coverage data from all your CI